community/gill-jito-airdrop/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "airdrop-claim-template",
+  "scripts": {
+    "build": "next build",
+    "airdrop:setup": "cd anchor && npx ts-node scripts/index.ts setup",
+    "airdrop:init": "cd anchor && npx ts-node scripts/index.ts initialize",
+    "airdrop:extract": "cd anchor && npx ts-node scripts/extract-wallet.ts --list",
+    "codama:generate": "cd anchor && anchor build && npx ts-node codama.config.ts",
+    "test": "cd anchor && vitest run tests/solana-distributor-comprehensive.test.ts",
+    "test:watch": "cd anchor && vitest tests/solana-distributor-comprehensive.test.ts",
+    "ci": "npm run build && npm run lint && npm run format:check",
+    "dev": "next dev --turbopack",
+    "format": "prettier --write .",
+    "format:check": "prettier --check .",
+    "lint": "next lint",
+    "start": "next start",
+    "type-check": "tsc --noEmit",
+    "type-check:node": "tsc --project tsconfig.node.json --noEmit"
+  },
+  "description": "Next.js, Tailwind, Wallet UI",
+  "keywords": [
+    "nextjs",
+    "react",
+    "tailwind",
+    "typescript",
+    "wallet-ui"
+  ],
+  "version": "0.0.0",
+  "private": true,
+  "engines": {
+    "node": ">=22.0.0",
+    "pnpm": ">=8.0.0"
+  },
+  "dependencies": {
+    "@coral-xyz/anchor": "^0.31.1",
+    "@solana/kit": "^2.3.0",
```

Comment:
We should be able to get rid of `@solana/kit` and import everything from `gill` instead.